useEvalGoICP: false
makeGoICPTest: false
maxTestIter: 2
noiseVariance: 0.001
GoICPdownsample: "1000" # demo (data_bunny -> model_bunny) if this value is set to 900 or below



